src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -81,6 +81,16 @@ private String getTransactionDisplayName(String transactionName) {
 		}
 	}
 	
+	static private void println() {
+		System.out.println();
+		System.out.flush();
+	}
+	
+	static private void println(Object obj) {
+		System.out.println(obj);
```

Comment:
flush is should be called after each print, useful when concurrent thread send output